node/pkg/watchers/aptos/watcher.go

```diff
@@ -36,57 +40,65 @@ type (
 )
 
 var (
-	aptosMessagesConfirmed = promauto.NewCounter(
+	aptosMessagesConfirmed = promauto.NewCounterVec(
```

Comment:
might be worth mentioning somewhere that this will change the prometheus reporting format for aptos